resources/keywords/import_provider_account.resource

```diff
@@ -39,12 +39,16 @@ User Navigates To Import Database Provider Account Screen From Database Access P
     ...        Will be removed once issue is fixed
 
 Select ${option} From ${menu} Button DropDown
-    ${btnMenuDropDownUpd}=    Replace String    ${btnMenuDropDown}    <<menu>>    ${menu}
-    Wait Until Keyword Succeeds    10s    1s    Wait Until Element Is Visible    ${btnMenuDropDownUpd}
-    Click Element    ${btnMenuDropDownUpd}
-    ${btnMenuOptDropDownUpd}=    Replace String    ${btnMenuOptDropDown}    <<menuopt>>    ${option}
-    Element Should Be Visible    ${btnMenuOptDropDownUpd}
-    Click Element    ${btnMenuOptDropDownUpd}
+    Wait Until Keyword Succeeds    10s    1s    Wait Until Element Is Visible    ${btnMenuDropDown}
```

Comment:
@ikanias Do we need this change? The earlier one was dynamic and capable of selecting dropdown values from given menu option like 
"Select _**Import Database Provider Account**_ From **_Configure_** Button DropDown"
"Select _**Create Database Instance**_ From **_Configure_** Button DropDown"

Reply:
The previous one is not working anymore and the tests are failing over it.

Reply:
@ikanias Can you please review again, As the keyword "Select ${option} From ${menu} Button DropDown" updated dynamically and it worked as expected. Attached the screenshot
![Screenshot from 2023-03-29 16-11-44](https://user-images.githubusercontent.com/86094767/228509527-dfab4741-15f2-4aa9-9a62-92b177bb6cc7.png)

Reply:
Strange... Sure will try again

Reply:
By the way it worked for me too before even when I removed these lines:
${btnMenuOptDropDownUpd}=    Replace String    ${btnMenuOptDropDown}    <<menuopt>>    ${option}
    Element Should Be Visible    ${btnMenuOptDropDownUpd}
    Click Element    ${btnMenuOptDropDownUpd}
So probably these commands(the ones I removed) are either redundant or not used.
The function 'Select {option} from {menu} Button DropDown' worked for me as well